src/main/java/io/epirus/console/project/InteractiveOptions.java

```diff
@@ -159,19 +156,11 @@ public boolean overrideExistingProject() {
         return userAnswer.toLowerCase().equals("y");
     }
 
-    public boolean userHasEpirusAccount() throws IOException {
-        if (LocalWeb3jAccount.configExists()) {
-            ObjectNode objectNode = LocalWeb3jAccount.readConfigAsJson();
-            return LocalWeb3jAccount.loginTokenExists(objectNode);
-        }
-        return false;
-    }
-
-    public boolean configFileExists() {
-        return LocalWeb3jAccount.configExists();
+    public boolean userIsLoggedIn() {
+        return config.getClientId() != null && config.getClientId().length() > 0;
     }
 
-    public boolean userWantsEpirusAccount() throws IOException {
+    public boolean userWantsEpirusAccount() {
```

Comment:
nit: rename method to `shouldCreateEpirusAccount`

Reply:
or `doesUserWantAnEpirusAccount`

i.e. make more semantically a `yes` or `no` question, for clarity